CONTRIBUTING.md

```diff
@@ -102,6 +103,7 @@ media, slack, newsletters, and email lists. You can also [reach us by
 email][contact].
 
 [repo]: https://example.com/FIXME
+[repo-issues]: https://example.com/FIXME/issues
```

Comment:
@JCSzamosi should we fix the links to point to the repo and repo-issues rather than https://example.com/FIXME?

Reply:
@henrykironde yep, that's the idea! You can do it as a commit to this PR or merge the PR and then open a new one for that change, whatever you prefer

Reply:
Here's the suggested change you can add as a commit to this PR

```suggestion
[repo-issues]: https://github.com/swcarpentry/sql-novice-survey/issues
```